pandas/conftest.py

```diff
@@ -522,7 +522,10 @@ def multiindex_dataframe_random_data(
     """DataFrame with 2 level MultiIndex with random data"""
     index = lexsorted_two_level_string_multiindex
     return DataFrame(
-        np.random.randn(10, 3), index=index, columns=Index(["A", "B", "C"], name="exp")
+        np.random.randn(10, 3),
+        index=index,
+        columns=Index(["A", "B", "C"], name="exp"),
+        copy=False,
```

Comment:
Is the False "needed" here (did it otherwise give failures), or just for efficiency since this is an example case where we know the array is not owned by anyone else?

Reply:
Copying here causes one test to fail, which is very weird(the failure). Haven't looked closer yet, but the test is useless as soon as your read_only pr is merged. 

Want to understand what's off there nevertheless though